test/datetime.cc

```diff
@@ -142,7 +142,7 @@ void TryIsoDateTime() {
             current_date_time)
       << std::string("Iso date time failed ") + current_date_time;
 
-  tz = DateTime::get_tz_db().from_index(DateTime::get_tz_db().to_index("Africa/Porto-Novo"));
+  tz = DateTime::get_tz_db().from_index(DateTime::get_tz_db().to_index("Africa/Lagos"));
```

Comment:
this was a long deprecated timezone which hasn't made its way into any tiles in the past 7 years